### ISSUE TITLE
`The following cops were added to RuboCop, but are not configured` message breaks json output

### ISSUE THREAD

**Issue by lstrzebinczyk** [rubocop/rubocop#9990]
Under proper circumstances rubocop shows `The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file.` warning. The warning is present even when `bundle exec rubocop -f json` is used, resulting in output being first, the message, and only then the json output. 

This breaks the `vscode-ruby-rubocop` VS Code extension and stops it from working, since [it asks for a json format](https://github.com/misogi/vscode-ruby-rubocop/blob/master/src/rubocop.ts#L179), and then [attempts to parse it](https://github.com/misogi/vscode-ruby-rubocop/blob/master/src/rubocop.ts#L234).

This behavior is present in 1.18.4  version of rubocop, which is the latest at the time of writing the issue.

--------

## Expected behavior

`bundle exec rubocop -f json` returns a valid json

## Actual behavior

`bundle exec rubocop -f json` does not return a valid json. It returns a textual warning, and only then a json.

## Steps to reproduce the problem

I created a reproduction repo at https://github.com/lstrzebinczyk/rubocop-json-format-issue.

Please note that the way the issue is reproduced is a little unorthodox, that's because if I understand it properly, the VS Code extension (mentioned earlier) is opening a [child process](https://github.com/misogi/vscode-ruby-rubocop/blob/master/src/rubocop.ts#L221), and reading the full output from it. 

It is also possible to reproduce the actual issue at hand by using vs code text editor, I'll happily provide reproduction steps for that if necessary. For simplicity of solution I decided to create a reproduction repo with equivalent (to the best of my understanding) issue.

## RuboCop version

```
$ bundle exec rubocop -V
1.18.4 (using Parser 3.0.2.0, rubocop-ast 1.9.0, running on ruby 2.6.5 x86_64-linux)
```

**Comment by koic** [comment]
I'm not sure about how best to resolve it for the json format, but this warning can be suppressed with the command line options or settings in .rubocop.yml shown below. In other words, it decides to enable or disable the pending cops.
https://docs.rubocop.org/rubocop/1.8/versioning.html#pending-cops

You can also use `bundle exec rubocop -f json 2> /dev/null` to suppress standard error output if you're using a Unix-like OS.

**Comment by sl4vr** [comment]
@lstrzebinczyk  I've tried to reproduce the issue on the repository you provided. That VS Code extension parses json only in `stdout` and checks for warnings in `stderr`, so no changes from Rubocop are required. I've managed to run extension and get Rubocop warnings in VS Code warning popup:

<img width="447" alt="Screenshot 2021-09-01 at 17 40 37" src="https://user-images.githubusercontent.com/11721556/131692465-f423106f-4d18-493f-beca-1265daaefcc7.png">


I think the issue might be in ruby executable version or location. As author mentioned in description:
> This extension may have problems when using a rvm or chruby environment. We recommend vscode-ruby. It can also lint ruby code.
